Tribler/Core/Video/VideoUtility.py

```diff
@@ -1,181 +0,0 @@
-# Written by Egbert Bouman
```

Comment:
We are not using any of this and if we use it, it should be implemented in the new GUI.